examples/Api.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import metadata from 'component-metadata!react-big-calendar/Calendar';
-import transform from 'lodash/object/transform';
+import { transform } from 'lodash/object';
 
```

Comment:
You are correct about the larger bundle size. I would suggest including [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash) (an officially supported plugin), which automatically modularizes the build size, regardless of whether using the long, manual includes, or the ES6 style, destructured imports that I switched to in this PR.